Dockerfile

```diff
@@ -5,9 +5,15 @@ MAINTAINER Thingpedia Admins <thingpedia-admins@lists.stanford.edu>
 # install packages
 USER root
 WORKDIR /opt/genienlp/
-ARG DECANLP_VERSION=master
+ARG GENIENLP_VERSION=master
 RUN pip3 install --upgrade pip
-RUN git fetch && git checkout ${DECANLP_VERSION} && pip3 install -e .
+RUN git fetch && git checkout ${GENIENLP_VERSION} && pip3 install -e .
+
+
+# download additional embeddings
+RUN genienlp cache-embeddings -d /usr/local/share/genienlp/embeddings --embeddings bert-base-multilingual-uncased
```

Comment:
This should probably be in the base image instead

Reply:
will move it then.